src/components/grid-aware/Spacer/Spacer.js

```diff
@@ -0,0 +1,29 @@
+import PropTypes from "prop-types";
+import React from "react";
+
+import s from "./Spacer.module.css";
+
+// color should be a value from global.css
+
+const Spacer = ({ heightDesktop, heightMobile, color }) => (
+  <div
+    style={{
+      "--spacer-height-desktop": `${heightDesktop}`,
```

Comment:
Please don't use string variable interpolation in a string that doesn't have anything else in it. You can just directly use the variable:

```suggestion
      "--spacer-height-desktop": heightDesktop,
```

Reply:
ok